sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -91,8 +91,11 @@ WITH base AS (
         ping_info.end_time,
         ping_info.parsed_end_time,
         ping_info.ping_type
-      ) AS ping_info,
+      ) AS ping_info
+      {% if not metrics_as_struct %}
+      ,
```

Comment:
bigquery sql is fine with trailing commas https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#trailing_commas

Reply:
Yeah, but in the SELECT statement. This is inside of REPLACE (...) and is not allowed (and painful to read).